components/feature/pwa/src/main/java/mozilla/components/feature/pwa/feature/WebAppSiteControlsFeature.kt

```diff
@@ -143,7 +143,7 @@ class WebAppSiteControlsFeature(
         } else {
             @Suppress("Deprecation")
             Notification.Builder(applicationContext).apply {
-                setPriority(NotificationCompat.PRIORITY_MIN)
```

Comment:
nit: Is not always better to use the `NotificationCompat` version?

Reply:
If your builder is also a compat builder yes. But here in this part of the code it's a normal builder and then lint doesn't seem to like if you use the compat class.